ak_vendor/report/report.py

```diff
@@ -405,6 +411,7 @@ def from_json(cls, data):
                 for owaspmobile2024 in data.get("owaspmobile2024", [])
             ],
             pcidss=[PCIDSS(**pcidss) for pcidss in data.get("pcidss", [])],
+            pcidss4=[PCIDSS4(**pcidss) for pcidss in data.get("pcidss4", [])],
```

Comment:
Please change the loop from `PCIDSS4(**pcidss) for pcidss in data.get("pcidss4", [])]` to `PCIDSS4(**pcidss4) for pcidss4 in data.get("pcidss4", [])]`